wiki/en/en-Installation-for-Android.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: wiki
+title: "Installation for Android"
+lang: "en"
+permalink: "/wiki/Installation-for-Android"
+---
+
+# Installation for Android
+
+Make sure you've already read the [Getting Started](Getting-Started) page.
+
+## Things to note about Android
+
+Although you **can** install Jamulus on Android devices (and hear sound), we strongly recommend **not** doing so. Sound quality - especially over WiFi - is usually bad and latency is high. If you have don't own a PC, we suggest you to buy a [Raspberry Pi](https://www.raspberrypi.org/){: target="_blank" rel="noopener noreferrer" } which is an inexpensive and small device that performs very well with Jamulus. Android support is just a proof of concept.
+
+## Install the Android PoC
+
+If you do want to try Jamulus on Android:
+
+1. Allow the installation of apps from unknown sources (look in Settings>Security).
+1. Download the APK from the [GitHub release page](https://github.com/corrados/jamulus/releases/tag/latest)
```

Comment:
after download, the apk needs to be installed. In my case: open list of downloads in chrome, klick on the filename of APK, confirm that I want to install.